From 2b01f304971284adecbd4705bddb06627cd72b7d Mon Sep 17 00:00:00 2001
From: Thomas Koeller <thomas.koeller@baslerweb.com>
Date: Thu, 5 Nov 2020 11:45:25 +0100
Subject: [PATCH 17/24] Reformatted Makefile

Reformatted the DTS Makefile for uniform coding style. This change is
purely cosmetic.

Applied from https://github.com/basler/meta-basler-imx8/tree/b1b6d6653931303bffbad727d767fe5a8d4844fc/recipes-kernel/linux/linux-imx

Signed-off-by: Alexander Stein <alexander.stein@ew.tq-group.com>
---
 arch/arm64/boot/dts/freescale/Makefile | 18 +++++++-----------
 1 file changed, 7 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index d85909b03148..2fcab221d52b 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -260,15 +260,11 @@ dtb-$(CONFIG_ARCH_MXC) += imx8ulp-evk.dtb imx8ulp-evk-rk055hdmipi4m.dtb \
 
 dtb-$(CONFIG_ARCH_S32) += s32v234-evb.dtb \
 			  s32v234-sbc.dtb
-
-
-dtb-$(CONFIG_ARCH_MXC) += \
-	imx8mm-evk-basler-camera.dtb \
-	imx8mm-ddr4-evk-basler-camera.dtb \
-	imx8mm-evk-basler-camera-1lane.dtb \
-	imx8mm-ddr4-evk-basler-camera-1lane.dtb
-dtb-$(CONFIG_ARCH_MXC) += \
-	imx8mq-evk-basler-camera.dtb \
-	imx8mq-evk-basler-camera-csi1.dtb \
-	imx8mq-evk-basler-camera-csi2.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mm-evk-basler-camera.dtb \
+			  imx8mm-ddr4-evk-basler-camera.dtb \
+			  imx8mm-evk-basler-camera-1lane.dtb \
+			  imx8mm-ddr4-evk-basler-camera-1lane.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mq-evk-basler-camera.dtb \
+			  imx8mq-evk-basler-camera-csi1.dtb \
+			  imx8mq-evk-basler-camera-csi2.dtb
 
-- 
2.25.1

